tests/fast-integration/test/1-commerce-mock.js

```diff
@@ -57,6 +60,12 @@ function commerceMockTests() {
       await checkFunctionResponse(testNamespace);
     });
 
+    it("should add, update and delete a service", async function () {
```

Comment:
Cool! The only thing is I'd move it after the lambda creation not to break the order of the main flow